src/app/components/forms/dynamic-form/terra-dynamic-form.component.ts

```diff
@@ -121,14 +123,21 @@ export class TerraDynamicFormComponent implements OnInit, OnChanges
     {
         if(!isNullOrUndefined(this.inputFormFunctions.onValueChangedCallback))
         {
-            this._formFieldControlService
+            let stream:Observable<any> = this._formFieldControlService
```

Comment:
Rename stream to stream$.

Actually the $-suffix is not part of the coding conventions, but we highly recommend it.